client/rest-client/src/main/java/org/infinispan/client/rest/RestCacheClient.java

```diff
@@ -16,8 +16,39 @@
     */
    String name();
 
+   /**
+    * Stream all keys
+    *
+    * @return Response with all keys streamed
+    */
    CompletionStage<RestResponse> keys();
 
+   /**
+    * Stream keys putting a limit
```

Comment:
Please see my comments review below @oraNod. I think it'd be more clear to use ```retrieve``` instead of ```stream``` in the API docs, clarifying the response contains an InputStream to consume the result

Reply:
@gustavonalle ack. I'll edit my review.